app/controllers/admin/users_controller.rb

```diff
@@ -4,8 +4,9 @@ def show
   end
 
   def index
+    quantity = 10
```

Comment:
nope, need to move to admin application controller like a constant DEFAULT_PER_PAGE=10